studio/src/pages/claimants/components/ClaimantForm.js

```diff
@@ -35,13 +41,22 @@ const ClaimantForm = ({ onCreate, data = {} }) => {
     });
   };
 
+  const getJsonVal = (val) => {
+    let regex = /,(?!\s*?[{["'\w])/;
+    let formattedJson = val.replace(regex, '');
+    return JSON.parse(formattedJson);
+  };
+
```

Comment:
Function `getJsonVal` looks repeated in all form components. Add that function in utils.